test/selectors/dashboard-selectors_spec.js

```diff
@@ -60,23 +64,44 @@ describe('dashboard selectors', () => {
     })
   })
 
-  describe('getSortedStudents', () => {
-    describe('when sortBy = "name"', () => {
-      const state = fromJS({
-        report: {
-          students: { 1: s1, 2: s2, 3: s3 }
-        },
-        dashboard: {
-          sortBy: 'name'
+  describe('getStudentTotalProgress', () => {
+    it('should return hash with student total progress', () => {
+      expect(getStudentTotalProgress(state({})).toJS()).to.eql(
+        {
+          1: 1.5,
+          2: 1.5,
```

Comment:
I'd rather normalize total progress to value from [0, 1] range. If we ever need to render some general progress bars, it will be easier to have this kind of value. I guess it should be very easy, just divide this value by number of activities if I understand things correctly.